### DISCUSSION TITLE
Make Vitruv independent of eclipse

### DISCUSSION THREAD

**Discussion by TomWerm** [vitruv-tools/.github#2] [Ideas]
The purpose of this discussion is to collect information about the dependencies on eclipse.

## Maven Build

* The current build process uses Tycho and builds OSGI artifacts. In the future, the build will only be done with maven. 
* To achieve this, the MANIFEST files must be replaced with poms that resolve dependencies via Maven (Central) instead of the eclipse updatesites. The deployment on the updatesite will be removed and replaced with a deployment on Central.
* Tycho will be removed
* Start with the test migration of one project, merge only after all projects have been migrated 

## Dependencies EmfBuilder / UI 

* Pull out to own repository (emf.ui, emf.ui.tests, vsum.ui) and build only framework CI against it

**Comment by arlange** [comment]
Due to our move to Maven, this discussion can be closed! Thanks for that!
